.github/workflows/generate-send-dependencies.yml

```diff
@@ -0,0 +1,26 @@
+name: Generate and submit dependency graph for language-server
+on:
+  push:
+    branches: ['main']
+
+permissions:
+  contents: write
+
+jobs:
+  dependency-submission:
+    runs-on: ubuntu-latest
+   
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-java@v4
+      with:
+        distribution: temurin
+        java-version: 21
```

Comment:
Would it be better to use Java 17 since the language server is built with that version? Would that affect the vuln analysis?

Reply:
Good point, it is better to use the same version.